lib/osf-components/addon/components/osf-layout/registries-side-nav/component.ts

```diff
@@ -31,4 +42,24 @@ export default class RegistriesSideNav extends Component {
     toggle() {
         this.toggleProperty('shouldCollapse');
     }
+
+    @action
+    delete() {
+        this.set('deleteEdit', false);
+        const node = this.toDelete;
```

Comment:
Where do we set `this.toDelete` in the first place?

Reply:
Hey, after our discussion this morning this a lot more sense. I will wait to discuss with you later before removing this, to make sure the changes are how you envision.